frontend/packages/core/package.json

```
@@ -37,7 +37,7 @@
     "js-cookie": "^2.2.1",
     "jwt-decode": "^2.2.0",
     "lodash": "^4.17.15",
-    "material-table": "^1.60.0",
+    "material-table": "1.60.0",
```

you can pin up to 1.68.0 i think
```suggestion
    "material-table": "1.68.0",
```

---

Done in the commit below to upgrade the lockfile